onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -6014,10 +6014,93 @@ def aten_var(self: TensorType, unbiased: bool = True) -> TensorType:
     raise NotImplementedError()
 
 
-def aten_var_mean(self: TensorType, unbiased: bool = True) -> tuple[TensorType, TensorType]:
+@torch_op("aten::var_mean", trace_only=True)
+def aten_var_mean(self: TReal, unbiased: bool = True) -> tuple[TReal, TReal]:
     """var_mean(Tensor self, bool unbiased=True) -> (Tensor, Tensor)"""
 
-    raise NotImplementedError()
+    if unbiased:
+        correction = 1
+    else:
+        correction = 0
+    var, mean = _aten_var_mean(self, correction, keepdim=False)
+    return var, mean
+
+
+@torch_op("aten::var_mean", overload=True, trace_only=True)
+def aten_var_mean_dim(
+    self: TReal, dim: int, unbiased: bool = True, keepdim: bool = False
+) -> tuple[TReal, TReal]:
+    """var_mean.dim(Tensor self, int[1]? dim, bool unbiased=True, bool keepdim=False) -> (Tensor, Tensor)"""
+
+    if unbiased:
+        correction = 1
+    else:
+        correction = 0
+    var, mean = _aten_var_mean_dim(self, dim, correction, keepdim)
+    return var, mean
+
+
+@torch_op("aten::var_mean", overload=True, trace_only=True)
+def aten_var_mean_correction(
+    self: TReal, dim: int = None, correction: int = 1, keepdim: bool = False
+) -> tuple[TReal, TReal]:
+    """var_mean.correction(Tensor self, int[1]? dim=None, *, Scalar? correction=None, bool keepdim=False) -> (Tensor, Tensor)"""
+
+    if correction is None:
+        correction = 1
+
+    if dim is None:
+        var, mean = _aten_var_mean(self, correction, keepdim)
+    else:
+        var, mean = _aten_var_mean_dim(self, dim, correction, keepdim)
+    return var, mean
+
+
+@torch_op("aten::var_mean", private=True)
```

Comment:
As we discussed before, let's use "internal" as the namespace of such function which was set private=True.

Reply:
I think private=True and "internal" are the same meaning.  Do we already have internal attribute for decorator?

Reply:
Discussed again in teams. The current name as is is good